File: dev/sbt-plugin/src/sbt-test/sbt-plugin/run-all-javadsl/b/api/src/main/java/api/BarService.java
package api;

import akka.stream.javadsl.Source;

import akka.NotUsed;
import com.lightbend.lagom.javadsl.api.ServiceCall;
import com.lightbend.lagom.javadsl.api.Descriptor;
import com.lightbend.lagom.javadsl.api.Service;
import com.lightbend.lagom.javadsl.api.transport.Method;
import static com.lightbend.lagom.javadsl.api.Service.*;

public interface BarService extends Service {

  ServiceCall<NotUsed, String> bar();

  @Override
  default Descriptor descriptor() {
    return named("/b").withCalls(restCall(Method.GET,  "/bar",    this::bar)).withAutoAcl(true);
  }
}

File: dev/sbt-plugin/src/sbt-test/sbt-plugin-commands/run-all-scaladsl/b/api/src/main/scala/api/BarService.scala
package api

import akka.NotUsed
import com.lightbend.lagom.scaladsl.api._
import com.lightbend.lagom.scaladsl.api.transport.Method

trait BarService extends Service {

  def bar: ServiceCall[NotUsed, String]

  override def descriptor = {
    import Service._
    named("b").withCalls(
      restCall(Method.GET, "/bar", bar)
    ).withAutoAcl(true)
  }
}